app/src/components/view/ranking/RankingListToday.tsx

```diff
@@ -63,13 +63,26 @@ const RankingListToday: React.FC<{ selectedTopic: number }> = ({
 			}
 		};
 
-		fetchData(getDate());
+		if (selectedTopic !== 0) {
+			fetchData(getDate());
+		}
 	}, [selectedTopic, getDate]);
 
 	if (isLoading) {
 		return <LoadingSpinner />;
 	}
 
+	if (!Array.isArray(data) || data.length === 0) {
+		return (
+			<div
+				className="flex items-center justify-center h-40 text-lg text-orange-500"
+			>
+				まだ投稿がありません！<br />
+				1位になれるかも!
+			</div>
+		);
+	}
```

Comment:
cameraページへ遷移するボタンも置いておくのはどうでしょうか？
(文字サイズと間隔も調整してみました。)

<img src="https://github.com/user-attachments/assets/8db7c1cc-d7ae-499f-b2c6-7953f4b0b431" width="30%" />

```suggestion
	if (!Array.isArray(data) || data.length === 0) {
		return (
			<div className="flex flex-col items-center justify-center h-full text-lg text-gray-700 mt-[20vh]">
				<p className="text-2xl font-bold mb-4">まだ投稿がありません</p>
				<p className="mb-4">今すぐ投稿して1位を目指しましょう！</p>
				<p className="text-gray-800 mb-4">
					あなたの投稿がトップに表示されるかも！
				</p>
				<Button
					variant="primary"
					className="flex items-center justify-center w-3/5 bg-gray-800 hover:bg-gray-700 text-white py-6 space-x-2 mt-4"
					onClick={() => router.push("/camera")}
				>
					<PhotoCameraIcon className="w-6 h-auto" />
					<span className="text-lg font-semibold">写真を撮る</span>
				</Button>
			</div>
		);
	}
```